INSTALL.md

```diff
@@ -93,12 +110,15 @@ $ make -f policy_utilities.mak clean
 $ make -f policy_utilities.mak
 ```
 
+## Building the Policy Generator - Setup Required
+
 If you wish to use the Policy Generator instead of manually composing the
-policies, you need to build it in utilities. The Policy Generator requires the
-json-schema-validator library which in turn depends on the JSON for Modern C++
-library.
+policies, you need to build the Generator in the `utilities/` directory.
+The Policy Generator requires the json-schema-validator library which in turn
+depends on the JSON for Modern C++ library.
 
-Follow these instructions to have them built and installed on your system:
+Follow these instructions to have these software packages built and installed
+on your system:
```

Comment:
Note to myself: Xref the new script in `CI/scripts`, as it packages all these steps in one script.

Reply:
Done. Updated this md to xref the CI-script.